<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecolationDialog</class>
 <widget class="QDialog" name="DecolationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decolation</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Separator</string>
    </attribute>
    <widget class="QTableView" name="seperatorTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addSeperatorButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>88</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSeperatorButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>88</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>191</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Number Of Job End Pages</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="endJobPagesSpinBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Accounting</string>
    </attribute>
    <widget class="QTableView" name="accountingTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addAccountingButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>88</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteAccountingButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>88</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name Template:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileNameTemplate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>481</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter a name that will later be attached to a printer. This name will define how</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>481</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>the printer will behave when encountering end-of-job triggers.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Decolation Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="decolationNameComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
